src/main/java/com/ibm/watson/developer_cloud/text_to_speech/v1/Customizations.java

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.ibm.watson.developer_cloud.text_to_speech.v1;
+
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.ibm.watson.developer_cloud.http.HttpMediaType;
+import com.ibm.watson.developer_cloud.http.RequestBuilder;
+import com.ibm.watson.developer_cloud.http.ResponseConverter;
+import com.ibm.watson.developer_cloud.http.ServiceCall;
+import com.ibm.watson.developer_cloud.text_to_speech.v1.model.CustomTranslation;
+import com.ibm.watson.developer_cloud.text_to_speech.v1.model.CustomVoiceModel;
+import com.ibm.watson.developer_cloud.util.GsonSingleton;
+import com.ibm.watson.developer_cloud.util.RequestUtils;
+import com.ibm.watson.developer_cloud.util.ResponseConverterUtils;
+import com.ibm.watson.developer_cloud.util.ResponseUtils;
+import com.ibm.watson.developer_cloud.util.Validator;
+
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+/**
+ * The Customizations service is a part of the Text to Speech API that allows
+ * the creation of a customized dictionary of words and their pronunciation.
+ * Users may manually specify the pronunciation of special terms with foreign
+ * origins, personal or geographic names, and abbreviations and acronyms for
+ * example.
+ * 
+ * <p>Use {@code new TextToSpeech().getCustomizations()} to instantiate this
+ * service.</p>
+ * 
+ * @version v1
+ * @see <a href=
+ *      "http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/doc/text-to-speech/custom-intro.shtml">
+ *      Customizations</a>
+ *
+ */
+public class Customizations {
```

Comment:
You are adding a Customizations class that can do API calls and that's a problem because we only do API calls from the service class (`TextToSpeech`). This is a design decision that doesn't follow OOP practices but it's required because we charge users when they do an API Call so we want to make sure that the user knows when a method could incur a charge.
Imagine a scenario when we use OOP and do a:

```java
voice.getName()
```

How do you know without looking at the `src` that we will not charge you and translate that method to a `/v1/voices/{id}`?

Let's move the customization calls to `TextToSpeech`